examples/templates/react/src/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+
+ReactDOM.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
```

Comment:
@jansiegel Should we remove StrictMode? This will trigger a warning with our current wrapper and "UNSAFE" use

Reply:
I don't see any errors in development mode using React examples. Can you provide more info, maybe a screenshot from the console?

Reply:
https://github.com/handsontable/handsontable/issues/7603